src/blockdev.rs

```diff
@@ -14,6 +14,7 @@
 
 use anyhow::{anyhow, bail, Context, Result};
 use gptman::{GPTPartitionEntry, GPT};
+use mbrman::{MBRPartitionEntry, MBR};
```

Comment:
Ah neat, I wasn't aware of these crates.  Looks like a good thing to use in https://github.com/containers/bootc/pull/30